repository-sort/sorting-list/demo.js

```diff
@@ -0,0 +1,59 @@
+visualize({
+    auth: {
+        name: "joeuser",
+        password: "joeuser",
+        organization: "organization_1"
+    }
+}, function(v) {
+
+    var sortByTypes = ['uri', 'description', 'type', 'creationDate', 'updateDate', 'accessTime', 'popularity'];
+
+    var sortByElement = document.getElementById('sortBy');
+
+    sortByTypes.forEach(function(i) {
+        let option = document.createElement('option');
+        option.text = i;
+        option.label = i;
+        sortByElement.add(option)
+    });
```

Comment:
We have the `label` property option defined in the HTML file and the rest in the JavaScript file here. This is inconsistent and can be confusing. Please move all the `property` options to the HTML. In this case, the main part related to sorting by API of Visualize.js will be more clear and easier to read.